CHANGELOG.develop

```diff
@@ -648,6 +648,13 @@ Other:
   - Added =symbol-overlay= to the =spacemacs-navigation= layer
     (thanks to kenkangxgwe)
 - Key bindings:
+  - GNUs keybindings now include *unplugged* and *slave* variants. These are for
```

Comment:
Moved the changelog entries to the GNUS section:
https://github.com/syl20bnr/spacemacs/blob/develop/CHANGELOG.develop#gnus

And made some minor rewrites:
```
  - Added *unplugged* and *slave* variants. These are for offline usage and
    for having multiple gnus running, respectively.
    - ~SPC a g g~ gnus (normal). Not slave, plugged.
    - ~SPC a g o~ slave & unplugged.
    - ~SPC a g u~ unplugged (but not slave).
    - ~SPC a g s~ gnus slave (but plugged).
      (thanks to Spenser Truex)
  - Added message mode insert binding: ~SPC m i F~ for FLAME ON, following
    [[https://tools.ietf.org/html/rfc1855][RFC 1855]]. (thanks to Spenser Truex)
```

Reply:
These are moved in the next commit.